libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/service/EvcsService.java

```diff
@@ -120,6 +121,18 @@ public void storePendingVc(VerifiableCredential credential) throws EvcsServiceEx
                                 credential.getVcString(), PENDING_RETURN, null, OFFLINE)));
     }
 
+    public void updatePendingIdentity(String userId, String evcsAccessToken)
```

Comment:
Should this describe what it's being updated to - perhaps it should be something like `abandonPendingIdentity`?

Reply:
Done